src/System.Management.Automation/security/wldpNativeMethods.cs

```diff
@@ -48,16 +48,23 @@ private SystemPolicy()
         /// <returns>An EnforcementMode that describes the system policy.</returns>
         public static SystemEnforcementMode GetSystemLockdownPolicy()
         {
-            if (s_allowDebugOverridePolicy || (s_systemLockdownPolicy == null))
+            if (s_systemLockdownPolicy == null)
             {
                 lock (s_systemLockdownPolicyLock)
                 {
-                    if (s_allowDebugOverridePolicy || (s_systemLockdownPolicy == null))
+                    if (s_systemLockdownPolicy == null)
                     {
-                        s_systemLockdownPolicy = GetLockdownPolicy(null, null);
+                        s_systemLockdownPolicy = GetLockdownPolicy(path: null, handle: null);
                     }
                 }
             }
+            else if (s_allowDebugOverridePolicy)
```

Comment:
So here, on subsequent calls we only make the call for the hook?

Reply:
Yes. If `s_allowDebugOverridePolicy` is true, that means the first call to `GetLockdownPolicy` did make into `GetDebugLockdownPolicy` without returning earlier, namely there is no platform policy enforced. So subsequent calls will just exercise the test hook code, without having to call the `Wldp` and `AppLocker` apis again.

Reply:
Yup, I was making sure I understood the change.